pkgs/applications/office/rime-cli/default.nix

```
@@ -0,0 +1,23 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "rime-cli";
+  version = "0.0.3";
+
+  src = fetchFromGitHub {
+    owner = "puddinging";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-CI0Jva7oA/zUGatv+wCdByqbTBNQRw+4clr8IDKX6HQ=";
+  };
+
+  vendorHash = null;
+
+  meta = with lib; {
+    homepage = "https://github.com/puddinging/rime-cli";
+    changelog = "https://github.com/puddinging/rime-cli/releases/tag/v${version}";
+    description = "关于小狼毫输入法的一个小工具";
```

I don't know if there's a rule for that but shouldn't the description be in English?

---

that is what their github repo lists as the cli description, and I was hesitant to trust google translate:
```suggestion
    description = "A small tool about Xiaolanghao input method";
```

---

I suggest describing it as “A command line tool to add customized vocabulary for Rime IME”